jediepcserver.py

```diff
@@ -127,8 +124,8 @@ def jedi_create_environment(venv, safe=False):
             return jedienv
 
 jedi_script_wrapper = jedi.Script
-JEDI_VERSION = pkg_resources.parse_version(jedi.__version__)
-if JEDI_VERSION < pkg_resources.parse_version('0.16.0'):
+JEDI_VERSION = packaging.version.parse(jedi.__version__)
+if JEDI_VERSION < packaging.version.parse('0.16.0'):
```

Comment:
Why do you use packaging here and importlib below? packaging is an external package so if it's added here it will need to be added as a dependency in setup.py

Reply:
> Why do you use packaging here and importlib below?

deprecated `pkg_resources` includes methods for *semantic versioning* and *module handling*. [Use of pkg_resources is deprecated in favor of `importlib` .. ](https://setuptools.pypa.io/en/latest/pkg_resources.html) .. but the problem is that `importlib` does not have methods for *semantic versioning*  .. as we want to do a comparison of *semantic versions* here, we have to use `packaging`.

> packaging is an external package so if it's added here it will need to be added as a dependency in setup.py

Thanks for pointing out, I added it --> https://github.com/tkf/emacs-jedi/compare/eb6d5a8fa5af5d07d14acd3cc05772c6edde50e0..6c36a91cd0cec35d1714929d4e74aff2b23b9367